serialization.go

```diff
@@ -58,7 +58,11 @@ func transformOut[T any](ctx context.Context, ans T) (T, error) {
 
 	_, ok := any(ans).(OutTransformer)
 	if ok {
-		return ans, errors.New("OutTransformer must be implemented by a POINTER RECEIVER. Please read the [OutTransformer] documentation")
+		err := errors.New("OutTransformer must be implemented by a POINTER RECEIVER. Please read the [OutTransformer] documentation")
```

Comment:
I'm unsure to understand the formatting of OutTransformer here

I mean it's an error message, not a godoc or a Markdown file

Am I missing something?

Reply:
Yeah the only place this should really manifest is in a log server side. Maybe a good time to rethink it's shape.